solaris/solaris10/generate_wazuh_packages.sh

```diff
@@ -256,8 +256,8 @@ build(){
     echo "| Building |"
     echo "------------"
 
-    groupadd ossec
-    useradd -g ossec ossec
+    groupadd wazuh
+    useradd -g wazuh wazuh
```

Comment:
This change require a PR in this Documentation section: https://documentation.wazuh.com/current/installation-guide/wazuh-agent/wazuh_agent_package_solaris.html#uninstall